### DISCUSSION TITLE
How to intercept-build a file?

### DISCUSSION THREAD

**Discussion by sanbrother** [rizsotto/scan-build#135] [General]
Hi ^_^

As I know, intercept-build should be followed by some command like 'make'
But, can intercept-build accept build log file directly?

Something like: intercept-build full_build_log.txt

Thank you.

**Comment by rizsotto** [comment]
No, the intercept command only creates a compilation database. And you can pass that to scan-build, which will run the analyzers. 

A simple build log won't work unfortunately.
